### ISSUE TITLE
Add support for no-fill options

### ISSUE THREAD

**Issue by iamdebodon** [zefoy/ngx-color-picker#195]
Hi @sconix ,

I have a requirement where I need to show no fill options. As shown below:

![image](https://user-images.githubusercontent.com/38261932/61457819-c4901700-a986-11e9-864a-c0d3eaa9edf4.png)

Regards,
Debobrata Dutta

**Comment by bhaskerchari** [comment]
@iamdebodon ,

Yes I believe that would be an interesting and useful option to provide. I also have few usecasses of that sort where it might come handy. I believe for the same reason excel has provided the option.

**Comment by sconix** [comment]
I am not sure if I see this as a perfect fit for this library, have to think it bit more. Anyway I do not have time to add such feature, but if PR is provided I could see this kind of feature included.

In the meantime you always have the possibility to make your own dialog component where you use this library as inline and add other options. Also I think there are other libraries that allow to have more custom color picker dialogs.

**Comment by bhaskerchari** [comment]
@sconix ,

I don't see any issue having a "No Fill" option to support and it would be a nice to have feature.

However coming to the workarounds you mentioned:
I am afraid that the possibility you mentioned to make our own dialog component where we use this library as inline and add other options is not so easy and straightforward. 

Because the dialog is opened by clicking on the element which has "colorPicker" directive attached.
And I don't think its straightforward to embed color picker dialog in a custom dialog with the current implementation. Do you have any reference to support the point which would be helpful? 

Thanks,
Bhasker

**Comment by sconix** [comment]
Naturally that level of customization is not straightforward but its not hard either if using some other library such as material dialog component etc. With those you can make a menu open really easily and place the dialog inside the menu etc. 

And as said I could add it if PR is made I do see it somewhat general usage and not complicating the code side too much therefore suitable for this library. I just do not have time to add such feature myself unless we get the need in our application.
